### PR TITLE
Update 2022-05-31.md

### PR DESCRIPTION
**Pull request by ashleywolf**

### CODE REVIEW

File: newsletter/2022-05-31.md [hunk 1]
```diff
@@ -2,31 +2,39 @@
...
-* [Please add ospo jobs]
+* [Aiven: Engineering Manager](https://apply.workable.com/aiven/j/B4AB10D57E/)
+* [Aiven: Senior Redis Open Source Developer](https://apply.workable.com/aiven/j/7BFFF9FF7E/)
+* [Aiven: Senior ClickHouse Open Source Developer](https://apply.workable.com/aiven/j/F0FB31090E/)
+* [Aiven: Senior Apache Kafka Open Source Developer](https://apply.workable.com/aiven/j/252CC6A4BD/)
+* [Aiven: Senior Open-Source Observability Developer](https://apply.workable.com/aiven/j/8E871F87B3/)
+* [Aiven: Senior OpenSearch Open Source Developer](https://apply.workable.com/aiven/j/324DD23DFA/)
+* [Aiven: Senior MySQL Open Source Developer](https://apply.workable.com/aiven/j/9A6026F629/)
+* [Aiven: Senior Cassandra Open Source Developer](https://apply.workable.com/aiven/j/BDE517268D/)
+* [Aiven: Head / Ambassador of the Open Source Program Office](https://apply.workable.com/aiven/j/AEE298EDEE/)
```
Comment (jlprat) [inline newsletter/2022-05-31.md:28]: This one is being rewritten, will be up in short!

**Review by jlprat — COMMENTED** [review]
We can add the EMEA & APAC working hours as well

File: newsletter/2022-05-31.md [hunk 2]
```diff
@@ -39,15 +39,12 @@ A list of OSPO related jobs published during this month!
 
 Here’s what you missed! A summary of the most recent OSPO related resources:
 
-* Post: [An Introduction to Open Source Program Offices (OSPO): a Community Conversion](https://semasoftware.medium.com/an-introduction-to-open-source-program-offices-ospo-a-community-conversion-90bfd05092e)
-* Article: [Spotify dances to the open source beat](https://venturebeat.com/2022/04/22/spotify-dances-to-the-beat-of-open-source/)
-* Article: [How an OSPO Can Help Your Engineers Give Back to Open Source](https://thenewstack.io/how-an-ospo-can-help-your-engineers-give-back-to-open-source/)
```
Comment (jlprat) [inline newsletter/2022-05-31.md:36]: :+1:

File: newsletter/2022-05-31.md (hunk 1 above)
Comment (jlprat) [reply newsletter/2022-05-31.md:28]: You can now use this link! https://apply.workable.com/aiven/j/15E66763C3/

**Comment by anajsana** [comment]
@ashleywolf what is the status of this? is it ok to be merged? 🙂

**Review by jlprat — APPROVED** [review]
LGTM

**Pull request merged**